docker-compose.auth.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.0'
+
+services:
+  grafana:
+    container_name: 'iotopen-datasource'
+    build:
+      context: ./.config
+      args:
+        grafana_image: ${GRAFANA_IMAGE:-grafana-enterprise}
+        grafana_version: ${GRAFANA_VERSION:-9.5.3}
+    ports:
+      - 3000:3000/tcp
+    environment:
+      GF_LOG_LEVEL: "debug"
+      GF_LOG_CONSOLE_FORMAT: "json"
+      GF_AUTH_GENERIC_OAUTH_CLIENT_ID: "EDIT_ME"
+      GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET: "EDIT_ME"
+      GF_AUTH_GENERIC_OAUTH_TOKEN_URL: "https://lynx-dev.iotopen.se/api/v2/oauth2/token"
+      GF_AUTH_GENERIC_OAUTH_AUTH_URL: "https://lynx-dev.iotopen.se/oauth2/authorize"
+      GF_AUTH_GENERIC_OAUTH_API_URL: "https://lynx-dev.iotopen.se/api/v2/oauth2/userinfo"
```

Comment:
Change these URL:s to the correct for production

Reply:
Usually, when developing, one uses the dev environment.
So i thought it would be more convenient to just use the dev as standard here.
You sure you want the production URL here?

Reply:
Solved by adding example URLS instead